### ISSUE TITLE
Supplement suggestion to use MathML with new "intent" attribute

### ISSUE THREAD

**Issue by NSoiffer** [w3c/wcag#5093]
MathML is mentioned in [Technique G115: Using semantic elements to mark up structure](https://www.w3.org/WAI/WCAG21/Techniques/general/G115).

MathML 4 (moving to CR) introduces a new accessibility feature aimed as disambiguating a math notation: [author intent.](https://www.w3.org/TR/mathml4/#mixing_intent). With the `intent` (and `arg`) attribute, authors can disambiguate speech and tell screen readers what should be spoken for a notation without specifying the exact words.

Two key benefits:
* **Localization**:  Allows the intent to be spoken in different languages.
* **Personalization**:  Can be customized to specific user needs (e.g., learning disabilities vs. blindness, or expert vs. novice levels).

The Math WG believes it would be useful to reference this feature, potentially with a simple example, within this section of Technique G115.

If the group agrees, I can craft a sentence or two and a simple example for inclusion.

**Comment by patrickhlauke** [comment]
> If the group agrees, I can craft a sentence or two and a simple example for inclusion.

It would be nice to get a separate isolated technique just for mathml perhaps?
